### ISSUE TITLE
Failed to connect to MPD server

### ISSUE THREAD

**Issue by nsirolli** [szclsya/mpdris2-rs#1]
When running mpdris2-rs the following error occurs:
```
nico@pilar:~> mpdris2-rs
ERROR [mpdris2_rs] Failed to connect to MPD server: missing fields from MPD status:
duration. Will try again every 5 secs...
```
I'm running arch, and I've got mpd running via a systemd user unit.

**Comment by szclsya** [comment]
Hi.

I've pushed a fix to allow empty duration (see 23e3e4754fba9a60a6fede6d4e294492e770681b). Have a try!

**Comment by nsirolli** [comment]
It works now, thanks!
